invokeai/frontend/web/src/features/nodes/components/flow/nodes/Invocation/fields/FieldLinearViewToggle.tsx

```diff
@@ -0,0 +1,74 @@
+import { IconButton } from '@invoke-ai/ui-library';
+import { createSelector } from '@reduxjs/toolkit';
+import { useAppDispatch, useAppSelector } from 'app/store/storeHooks';
+import {
+  selectWorkflowSlice,
+  workflowExposedFieldAdded,
+  workflowExposedFieldRemoved,
+} from 'features/nodes/store/workflowSlice';
+import { memo, useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { PiMinusBold, PiPlusBold } from 'react-icons/pi';
+
+type Props = {
+  nodeId: string;
+  fieldName: string;
+  isHovered: boolean;
+};
+
+const FieldLinearViewToggle = ({ nodeId, fieldName, isHovered }: Props) => {
+  const dispatch = useAppDispatch();
+  const { t } = useTranslation();
+
+  const selectIsExposed = useMemo(
+    () =>
+      createSelector(selectWorkflowSlice, (workflow) => {
+        return Boolean(workflow.exposedFields.find((f) => f.nodeId === nodeId && f.fieldName === fieldName));
+      }),
+    [fieldName, nodeId]
+  );
+
+  const isExposed = useAppSelector(selectIsExposed);
+
+  const handleExposeField = useCallback(() => {
+    dispatch(workflowExposedFieldAdded({ nodeId, fieldName }));
+  }, [dispatch, fieldName, nodeId]);
+
+  const handleUnexposeField = useCallback(() => {
+    dispatch(workflowExposedFieldRemoved({ nodeId, fieldName }));
+  }, [dispatch, fieldName, nodeId]);
+
+  const ToggleButton = useMemo(() => {
+    if (!isHovered) {
+      return null;
+    } else if (!isExposed) {
+      return (
+        <IconButton
+          mx="2"
```

Comment:
This adds an (what I perceive to be) some awkward space to the right of the button. 

I'm not sure what the intention is, but if it's to ensure there is space between the field title and button, we can use flex gap on in `InputField` instead.

With `mx={2}`:
<img width="360" alt="image" src="https://github.com/invoke-ai/InvokeAI/assets/4822129/be639d26-d63e-484e-a3be-ab3fb4068d24">

Without it:
<img width="388" alt="image" src="https://github.com/invoke-ai/InvokeAI/assets/4822129/891c4c8f-62e4-49e2-beb1-14dbd61e1242">

Reply:
Whoops, meant to remove this! Was there cause button used to be somewhere else

Reply:
> What do yall think about putting these more inline with the label, instead of on the other side of the container?

I think it's a better UX when the buttons are in the same position for every field - if they are just at the end of the text then they are different for every field. If they are _before_ the field title, then you have layout shift when you mouse over